packages/server/src/migrations/sync_app_and_reset_rows_quotas.js

```diff
@@ -0,0 +1,27 @@
+const { MIGRATIONS, MIGRATION_DBS, migrateIfRequired } =
+  require("@budibase/backend-core").migrations
```

Comment:
Could we replace this with the new standard method `("@budibase/backend-core/migrations")`? I've put together a [PR](https://github.com/Budibase/budibase/pull/4004) to standardise it completely across the usages in the monorepo with some explanation as to the reasoning and the stuff I tested.